doc/sphinx/language/cic.rst

```diff
@@ -1046,6 +1046,38 @@ between universes for inductive types in the Type hierarchy.
       exT_intro : forall X:Type, P X -> exType P.
 
 
+.. example:: Negative occurrence
+
+   The following inductive definition is rejected because it does not
+   satisfy the positivity condition:
+
+   .. coqtop:: all
+
+      Fail Inductive I : Prop := not_I_I (not_I : I -> False) : I.
+
+   If we were to accept such definition, we could derive a
+   contradiction from it (we can test this by disabling the
+   :flag:`Positivity Checking` flag):
+
+   .. coqtop:: none
+
+      Unset Positivity Checking.
+      Inductive I : Prop := not_I_I (not_I : I -> False) : I.
+      Set Positivity Checking.
+
+   .. coqtop:: all
+
+      Definition I_not_I '(not_I_I not_I : I) : ~ I := not_I.
+
+   .. coqtop:: in
+
+      Lemma contradiction : False.
+      Proof.
+      enough (I /\ ~ I) as [] by contradiction.
+      split.
+      apply not_I_I; intro; now apply I_not_I.
+      intro; now apply I_not_I.
+      Qed.
```

Comment:
I think this would be a bit more readable with indentation and bullets (and possibly by using an explicit match in `I_not_I`?), since the example is already sufficiently well-written to allow reading it without running it.

Reply:
I initially tried using bullets and indentation, but the rendering was very different from what I had written: indentation was not kept, and spurious newlines appeared.

OK for the explicit match, probably clearer indeed.

Reply:
> I had written: indentation was not kept, and spurious newlines appeared.

My oh my. Can you file a bug report and assign me?